.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
remove all of the files in the .vs folder

Reply:
done.